proposals/first-class-span-types.md

```diff
@@ -62,37 +62,46 @@ as if the `T` was declared as `out T` in some scenarios. We do not, however, plu
 variance-convertible in [§18.2.3.3](https://github.com/dotnet/csharpstandard/blob/draft-v8/standard/interfaces.md#18233-variance-conversion). If in the future, we change the runtime
 to more deeply understand the variance here, we can take the minor breaking change to fully recognize it in the language.
 
-Practically, this will also mean that in pattern matching for generic scenarios, we'd have behavior as follows:
+There is also an open question below about participation in delegate signature matching.
 
-```cs
-using System;
+#### Patterns
 
-M<object[]>(["0"]); // Does not print
-M<ReadOnlySpan<string>>(["1"]); // Does not print
-M<Span<object>>(["2"]); // Does not print
-M<ReadOnlySpan<object>>(["3"]); // Prints
+Note that `ref struct`s are not supported in patterns, except for trivial identity patterns:
```

Comment:
I don't think is quite correct. For example, this works:

```cs
Span<int> ints = [1, 2, 3];

if (ints is [1, 2, 3])
{
    Console.WriteLine("Here");
}
```

I think this needs to be more specific: When used as a `type` in any pattern, only identity conversions are allowed.